encoding/streams/readable-writable-properties.any.js

```diff
@@ -0,0 +1,22 @@
+// META: global=worker
+
+// This just tests that the "readable" and "writable" properties pass the brand
+// checks. All other relevant attributes are covered by idlharness.html.
```

Comment:
This is already out-of-date (it's idlharness.any.js), but it seems better to say "IDL tests" here given future refactoring in that area.